crawl-ref/source/dat/species/kobold.yaml

```
@@ -33,6 +33,11 @@ levelup_stat_frequency: 5
 mutations:
   1:
     MUT_CARNIVOROUS: 1
+    MUT_NIGHTSTALKER: 1
```

Having night stalker ramp up tilts the power balance to the lategame which is already true of so much of crawl. I would be interested in trying this with night stalker 3 from XL 1, or maybe the ramp up being much shorter.
```suggestion
    MUT_NIGHTSTALKER: 3
 ```
```